ddl/sequence_test.go

```diff
@@ -1088,3 +1090,17 @@ func (s *testSequenceSuite) TestAlterSequencePrivilege(c *C) {
 	c.Assert(err.Error(), Equals, "[planner:1142]ALTER command denied to user 'myuser'@'localhost' for table 'my_seq'")
 	tk.MustExec("drop sequence if exists my_seq")
 }
+
+func TestDdl_AlterSequenceIssue31265(t *testing.T) {
+	store, clean := testkit2.CreateMockStore(t)
+	defer clean()
+
+	tk := testkit2.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("create sequence seq cache=1 nocache")
```

Comment:
`cache=1` conflicts with `nocache`, why not report an error?

Reply:
the late one will take effect